src/adr/0051-email-rate-limiting.md

```diff
@@ -0,0 +1,40 @@
+# Email rate limiting
+
+Date: 2021-01-10
+
+## Status
+
+// todo
+
+## Context
+
+Rate limiting is one of the most popular features of any service. They limit how many emails you can send/receive from/to
+each email account over a given period of time.  
+We believe the rate-limiting will help James has more control of the resource and easy to dynamic config the user policy.
```

Comment:
Rate limiting can also be a security requirement for SaaS deployments. It also minimises impacts of Open-relay types of compromission.

Reply:
Also add "it complements storage quota".